easybuild/easyconfigs/l/LibSoup/LibSoup-2.65.2-fosscuda-2018b.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'MesonNinja'
+
+name = 'LibSoup'
+version = '2.65.2'
+
+homepage = 'https://wiki.gnome.org/Projects/libsoup'
+description = """libsoup is an HTTP client/server library for GNOME. It
+uses GObjects and the glib main loop, to integrate well with GNOME
+applications, and also has a synchronous API, for use in threaded
+applications."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['8435a721c52624fcfd20467767fdf1fab55c8ed1cb6f0a08b8a643456d5ed1a1']
+
+builddependencies = [
+    ('Meson', '0.48.1', '-Python-3.6.6'),
+    ('Ninja', '1.8.2'),
+    ('GObject-Introspection', '1.54.1', '-Python-2.7.15'),
```

Comment:
@akesandgren Shouldn't mix two `Python` versions...

Reply:
You're quite right, so there was a reason i had created GObj... for python-3.6.6...
It didn't exist previously. I'll add it to this one.